doc/man3/SSL_CTX_set_cert_verify_callback.pod

```diff
@@ -38,8 +38,9 @@ All values, including 0, are ignored
 if the verification mode is B<SSL_VERIFY_NONE>.
 Otherwise, when the return value is 0, the handshake will fail.
 
-In client mode I<callback> may also return -1,
-typically on failure verifying the server certificate.
+In client mode I<callback> may also call SSL_set_retry_verify() function
+on the B<SSL> object set in the I<x509_store_ctx> ex data (see
+L<SSL_get_ex_data_X509_STORE_CTX_idx(3)>.
 This makes the handshake suspend and return control to the calling application
 with B<SSL_ERROR_WANT_RETRY_VERIFY>.
```

Comment:
Will calling SSL_set_retry_verify() "make the handshake suspend and return control to the calling application
with B<SSL_ERROR_WANT_RETRY_VERIFY" _regardless_ of what value the verification callback _returns_ after calling SSL_set_retry_verify()?! Please adjust the documentation to clarify/codify the relationship between calling SSL_set_retry_verify(), the return value of the callback, and the handshake suspension.

The examples (provided elsewhere, thank you!) are great, but they do not tell the reader what happens in cases not shown in the examples.

Reply:
Fixed.